src/renderer/components/ft-profile-bubble/ft-profile-bubble.js

```diff
@@ -18,11 +18,19 @@ export default Vue.extend({
     textColor: {
       type: String,
       required: true
+    },
+    profileImageUrl: {
+      type: String,
+      required: true
     }
   },
   computed: {
     profileInitial: function () {
       return this?.profileName?.length > 0 ? Array.from(this.profileName)[0].toUpperCase() : ''
+    },
+    hasProfileImage: function () {
+      // the profile image url should both exist and be a non-empty string
+      return this?.profileImageUrl && this.profileImageUrl?.length > 0
```

Comment:
You have made the property required, so it will always be a string, so you don't need to check for null and undefined
```suggestion
      return this.profileImageUrl.length > 0
```